cfme/common/provider.py

```diff
@@ -3,6 +3,7 @@
 from manageiq_client.api import APIException
 
 import cfme
+import cfme.base.credentials
```

Comment:
Why import module here instead of just the Credential class? (Looks like the class is just used for inheritence here.